packages/components/ng-log-live-tail/src/controller.js

```diff
@@ -155,16 +168,53 @@ export default class LogLiveTailCtrl {
   }
 
   formatLogs(data) {
-    return data.messages.map((log) => {
-      return this.keys.reduce((customLog, key) => {
-        const path = this.logKeys[key];
+    return data.messages.map(({ message }) => {
+      const { _id, timestamp, level } = message;
+
+      // LOG PREFIX
+      const formattedTimestamp = format(timestamp, 'P pp', {
+        locale: this.dateFnsLocale,
+      });
+
+      const levelLabel = LEVEL_LABELS[level];
+
+      // LOG CUSTOM FIELDS
+      const customFields = this.logKeys.reduce((customLog, key) => {
         // eslint-disable-next-line no-param-reassign
-        customLog[key] = this.getValueByPath(log, path) || '';
+        customLog[key] = this.getValueByPath(message, key) || '';
         return customLog;
       }, {});
+
+      const formattedLog = {
+        _id,
+        formattedTimestamp,
+        level,
+        levelLabel,
+        ...customFields,
+      };
+
+      return formattedLog;
     });
   }
 
+  /**
+   * angularJS version of packages/manager/core/utils/src/datefns/index.ts
+   * Converts an ovh locale to date-fns locale.
+   * Examples:
+   *   getDateFnsLocale('fr_FR') => 'fr'
+   *   getDateFnsLocale('fr_CA') => 'frCA'
+   */
+  static getDateFnsLocale(ovhLocale) {
+    if (ovhLocale === 'en_GB') {
+      return 'enGB';
+    }
+    if (ovhLocale === 'fr_CA') {
+      return 'frCA';
+    }
+    const [locale] = ovhLocale?.split('_');
```

Comment:
Can you take a look to the sonarcloud alert `Unsafe usage of optional chaining. If it short-circuits with 'undefined' the evaluation will throw TypeError.`